general-guidelines/validating-api-descriptions.md

```diff
@@ -72,3 +70,36 @@ Currently any problem of severity of error or warning will cause a failure statu
 ```
 spectral lint my-api-spec.yaml --skip-rule=protocol-https-only-oas3
 ```
+
+## Spectral rules list
+
+The documentation on Spectral general OAS and specific rules for OAS2 and OAS3 can be found here: https://github.com/stoplightio/spectral/blob/develop/docs/reference/openapi-rules.md. 
+
+Adidas specific rules are listed below:
+
+### Adidas general rules
+
+* `paths-camelCase` - All YAML/JSON paths MUST follow camelCase.

```

Comment:
Just an idea: Shouldn't we prefix all adidas rules with something like `adi-` or `adidas-` to distinguish from the others? (Does not have to be in this PR)

Reply:
thats something to discuss i think